src/server/session.ts

```diff
@@ -367,76 +343,73 @@ namespace ts.server {
         return info && !info.isLocal ? { fileName: info.fileName, pos: info.textSpan.start } : undefined;
     }
 
-    function combineProjectOutputForReferences(
+    function getReferencesWorker(
         projects: Projects,
         defaultProject: Project,
-        initialLocation: DocumentPosition,
+        initialPosition: DocumentPosition,

```

Comment:
I could be persuaded to rename these back to location, since that seems to be a convention, but it doesn't make sense to me.